src/it/README.md

```diff
@@ -0,0 +1,94 @@
+has 4 identical test cases (with special tweaks only for Tomee), e.g. with Thorntail it is:
+
+    public void thorntailEmpty() {
+        testRuntime("THORNTAIL_V2", "thorntail",
+                SpecSelection.EMPTY, new int[]{9990});
+    }
+
+    public void thorntailAll() {
+        testRuntime("THORNTAIL_V2", "thorntail",
+                SpecSelection.ALL, new int[]{9990, 8180, 10090});
+    }
+
+    public void thorntailAllButJWTRest() {
+        testRuntime("THORNTAIL_V2", "thorntail",
+                SpecSelection.ALL_BUT_JWT_REST, new int[]{9990});
+    }
+
+    public void thorntailJWTRest() {
+        testRuntime("THORNTAIL_V2", "thorntail",
+                SpecSelection.JWT_REST, new int[]{9990, 8180, 10090});
+    }
+
+ * Empty test case does not select any examples, so only the default HelloWorld endpoint is tested.
+ * All selects all examples and tests them all (expect for Open Tracing as Jaeger is on TODO list).
+ * AllButJWTRest selects and tests all but JWT and REST; the main purpose is to ensure that no service-a/service-b division takes place.
+ * JWTRest selects and tests only JWT and REST.
+ 
+ Those additional ports are for the final cleanup after test, for a loop that awaits socket closure.
+ The main landing application port is parsed from README file, but there are other auxiliary ports that had to be hardcoded here.
+ 
+# How does it work
+REST API is used to download a zip. The zip file is unzipped.
+README file(s) are parsed to get the first mvn command,
+which is assumed to be the build one. The first java command from README is assumed to be the one to run the server.
+Multiple flavours of builds, i.e. multiple ways to build the application is an unsupported feature in the TS at the moment.
+Landing page address is also parsed from the README file as the first one beginning with http://.
+
+
+Build commands are used to build the generated project(s). If you have never run the TS, this might take a long time. Mind
+you are downloading all dependencies for THORNTAIL_V2, PAYARA_MICRO, LIBERTY, HELIDON, KUMULUZEE and TOMEE.
+
+Run commands are then used to run the application(s).
+
+A loop awaits them to be available and then each spec is tested, see MPSpec enum for definitions of expected content.
+
+Last but not least, build and runtime logs are checked for errors. See Whitelist enum for whitelisting known and expected error messages.
+
+At the very end of the execution, application(s) are terminated. There is also a loop that waits for a TCP socket to die.
+# Platforms
+The TS was tested on OpenJDK Java 11 J9 and HotSpot on Linux and with OpenJDK 11 HotSpot on Windows. With Windows,  ```powershell``` is used
+instead of ```unzip``` and ```taskkill``` instead of ```kill```. There is also some ```wmic``` heuristics to clean hanging processes.
+If you are running the TS on Mac, make sure your ```unzip``` can work with the arguments TS supplies or open a PR.
```

Comment:
Works OK on MAC

Reply:
Thank you for testing. Will update the readme.